README.md

```diff
@@ -76,6 +76,7 @@ Alternatively, you can import the library directly from a CDN:
 -   [checkPermissionStatus](#checkPermissionStatus)
 -   [internalNavigation](#internalNavigation)
 -   [dismiss](#dismiss)
+-   [matchContactData](#matchContactData)
```

Comment:
to me this is a bad name, match contact data? Why *match*? why *singular* contact if you can return many? data? everything is data. 

You are fetching contacts by phone number so:
requestContactsByPhone
fetchContactsByPhone
getContactsByPhone

This way, if in the future you want to fetch by email or name, you can follow this same convention

Reply:
I gave this name, because of the type of the bridge, that's why I choose that name, but you're  right and in this case I prefer requestContactsByPhone. @atabel what do you think?

Reply:
That's a great suggestion. I like fetchContactsByPhone. I will update the TS if everyone agrees.

Reply:
fetchContactsByPhone is the name function in the bridge, this is not the type MATCH_CONTACT_DATA, but If you prefer you can change the type.

Reply:
Changes done!